src/sentry/static/sentry/app/utils/performance/quickTrace/utils.ts

```diff
@@ -16,6 +17,21 @@ export function isTransaction(event: Event): event is EventTransaction {
   return event.type === 'transaction';
 }
 
+/**
+ * An event can be an error or a transaction. We need to check whether the current
+ * event id is in the list of errors as well
+ */
+function isCurrentEvent(
+  event: TraceFull | QuickTraceEvent,
+  currentEvent: Event
+): boolean {
```

Comment:
Should this only check the errors array for the current event depending on the current event's type? Seems like avoidable computation.